src/Symfony/Component/Routing/Tests/RouteCollectionTest.php

```diff
@@ -105,9 +105,9 @@ public function testAddCollection()
     public function testAddCollectionWithResources()
     {
         $collection = new RouteCollection();
-        $collection->addResource($foo = new FileResource(__DIR__.'/Fixtures/foo.xml'));
+        $collection->addResource($foo = new FileResource(__DIR__.'/Fixtures/empty.yml'));
         $collection1 = new RouteCollection();
-        $collection1->addResource($foo1 = new FileResource(__DIR__.'/Fixtures/foo1.xml'));
+        $collection1->addResource($foo1 = new FileResource(__DIR__.'/Fixtures/file_resource.yml'));
```

Comment:
I think those changes removing usage of XML fixtures for tests that are not related to the XML loader should be done in the 7.4 branch instead. This will avoid getting conflicts on those lines for merge ups during the 7.4 maintenance lifecycle for no benefit.